lib/matplotlib/artist.py

```diff
@@ -668,6 +668,17 @@ def set_sketch_params(self, scale=None, length=None, randomness=None):
         """
         Set the sketch parameters.
 
+        Requires the following preamble when using the PGF backend:
+
+            \\usepackage{pgf}
```

Comment:
This one is implied when using the PGF backend, isn't it?

Reply:
Yes, that was a leftover from the MWE in the bug report. I removed it.